client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -58,6 +66,14 @@ class TransferDomainPrecheck extends React.PureComponent {
 		this.props.setValid( domain, supportsPrivacy );
 	};
 
+	goToMapDomainStep = event => {
+		event.preventDefault();
+		page( paths.domainMapping( this.props.selectedSite.slug, this.props.domain ) );
+	};
+
+	transferIsRestricted = () =>
+		! this.state.loading && 'not_restricted' !== this.state.transferRestrictionStatus;
```

Comment:
Extract to a helper function.

Reply:
Since not making this global right now, leaving as is.